askama/Cargo.toml

```diff
@@ -1,11 +1,11 @@
 [package]
-name = "rinja"
+name = "askama"
 version = "0.3.5"
```

Comment:
```suggestion
version = "0.13.0"
```

Reply:
I suggest to do that in another PR. We'll need to update the version before the release in any case.